moto/packages/httpretty/core.py

```diff
@@ -103,6 +103,12 @@
 except ImportError:  # pragma: no cover
     ssl = None
 
+try:  # pragma: no cover
```

Comment:
Damn. Nice fix. It's sad that this kind of thing is necessary.

Reply:
Yeah it was a right pain